app/src/main/java/com/cmput301f20t21/bookfriends/ui/profile/ProfileFragment.java

```diff
@@ -47,6 +63,25 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         emailAddress = view.findViewById(R.id.email);
         phoneNumber = view.findViewById(R.id.phone);
         editProfile = view.findViewById(R.id.image_edit);
+        name = view.findViewById(R.id.username);
+        phoneNumber.setText("");
+
+        // get the login information from firebase
+        User firebaseUser = AuthService.getInstance().getCurrentUser();
+        if (firebaseUser != null) {
+            String userId = firebaseUser.getUid();
+            String username = firebaseUser.getUsername();
+            String email = firebaseUser.getEmail();
+            name.setText(username);
+            emailAddress.setText(email);
+
+            //once the profile is opened, user can edit their phone number, and new field in firebase is created
+            HashMap<String, Object> data = new HashMap<>();
+            data.put("phone", phoneNumber.getText().toString());
+            FirebaseFirestore.getInstance().collection("users").document(userId)
+                    .set(data, SetOptions.merge());
```

Comment:
I think this part of code is duplicate of the update logic in EditFragment (commented above) and should be removed. please double check on this part and maybe pick one.